src/commands/lambda/instrument.ts

```diff
@@ -483,6 +501,30 @@ export class InstrumentCommand extends Command {
     }
   }
 
+  private setEnvServiceVersion() {
+    const environment = process.env[ENVIRONMENT_ENV_VAR]
+    if (environment === '') {
+      // If a user doesn't enter something then inquirer will set the answer to an empty string and in that case we don't want to set these environment variables
+      this.environment = undefined
+    } else {
+      this.environment = environment
+    }
```

Comment:
Nit but checking against the empty string is also a bit brittle IMO, why if the library sets that to `undefined` or `null` in the future? I would treat everything the same by simply checking `if (!environment)`. Also in that case you can simply shorten to `this.environment = process.env[ENVIRONMENT_ENV_VAR]  || undefined`.

Same remarks for the two checks below.

Reply:
interesting, I didn't know this could do the same thing and I see how this is more robust, Thanks